components/inquiry_questionnaire/index.coffee

```diff
@@ -69,13 +74,17 @@ module.exports = ({ user, artwork, inquiry, bypass }) ->
 
     # End of complete flow
     .on 'done', ->
-      # Send the inquiry immediately
-      inquiry.send {},
-        success: ->
-          modal.close()
-        error: (model, response, options) ->
-          modal.close ->
-            openErrorFlash response
+
+      if inquiry.send?
```

Comment:
If the inquiry is an actual artwork inquiry, it will have a `send` method, otherwise we assume the inquiry has already been sent, so just close the modal.